src/WebJobs.Script/Utility.cs

```diff
@@ -144,6 +145,14 @@ public static string GetFunctionShortName(string functionName)
             return functionName;
         }
 
+        // "Namespace.Class.Method" --> "Namespace.Class"
+        public static string GetFullClassName(string fullFunctionName)
```

Comment:
Perhaps this should use the FunctionEntryPointResolver instead of doing a string manipulation?